src/main/java/com/commercetools/sync/producttypes/utils/ProductTypeUpdateAttributeDefinitionActionUtils.java

```diff
@@ -180,8 +170,7 @@
                     .map(attributeDefinitionDraft ->
                         buildActions(
                             oldAttributeDefinition,
-                            attributeDefinitionDraft,
-                            syncOptions
+                            attributeDefinitionDraft
```

Comment:
why not all on the same line :) ?
````java
.map(attributeDefinitionDraft -> buildActions(oldAttributeDefinition, attributeDefinitionDraft))
````

Reply:
this was due to there was an third parameter that mades the line being greater than 120, so after removing this third parameter I forgot to format it in one line

Reply:
1560ab147719ff45e48e57b7ff470a836f225d9d